utils/self-extracting-executable/decompressor.cpp

```diff
@@ -168,6 +168,24 @@ int decompress(char * input, char * output, off_t start, off_t end, size_t max_n
     return 0;
 }
 
+bool getSudoIDs(uid_t &sudo_uid, uid_t &sudo_gid)
```

Comment:
This is ok, but... why don't we set it the same as in the original file?
The invariant will be: we don't change the owner/group of the file while extracting.

Reply:
yes, makes sense